src/Client/src/app/supervisor-panel/management/supervisor-transactions/supervisor-transactions.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="container mt-3">
+    <div class="mt-5" *ngIf="loading || error">
+        <app-loading-spinner *ngIf="loading && !error"></app-loading-spinner>
+        <div class="alert alert-danger" role="alert" *ngIf="error && !loading">
+          {{ error }}
+        </div>
+    </div>
+    <div class="row">
+      <form [formGroup]="form">
+      <div class="mb-3 col-6 offset-3">
+          <label class="form-label" for="status">Status</label>
+          <select id="status" class="form-select mb-2" formControlName="status" (change)="onFilterByStatus()">
+              <option *ngFor="let status of statuses | keyvalue" [value]="status.key">{{status.value}}</option>
+          </select>
+      </div>
+    </form>
+  </div>
+    <div class="mat-elevation-z8 mb-5 table-responsive" *ngIf="loading === false">
+        <table mat-table [dataSource]="dataSource">
+            <ng-container matColumnDef="id">
+              <th mat-header-cell *matHeaderCellDef>Transaction ID</th>
+              <td mat-cell *matCellDef="let element"> {{ element.id }} </td>
+            </ng-container>
+  
+            <ng-container matColumnDef="category">
+              <th mat-header-cell *matHeaderCellDef> Category </th>
+              <td mat-cell *matCellDef="let element"> {{element.category | titlecase}} </td>
+            </ng-container>
+  
+            <ng-container matColumnDef="name">
+              <th mat-header-cell *matHeaderCellDef> Product </th>
+              <td mat-cell *matCellDef="let element"> {{element.name | titlecase}} </td>
+            </ng-container>
+
+            <ng-container matColumnDef="companyProfit">
+              <th mat-header-cell *matHeaderCellDef> Company profit </th>
+              <td mat-cell *matCellDef="let element"> {{element.companyProfit | currency}} </td>
+            </ng-container>
+  
+            <ng-container matColumnDef="date">
+              <th mat-header-cell *matHeaderCellDef> Date </th>
+              <td mat-cell *matCellDef="let element"> {{element.date | date}} </td>
```

Comment:
![image](https://user-images.githubusercontent.com/47317247/141753117-74a0a3a2-6988-4263-9afa-5100d92a2ca0.png)
Nie wyświetla daty

Reply:
Jaki masz język/region ustawiony w systemie? U mnie na US j
![Screenshot 2021-11-15 204828](https://user-images.githubusercontent.com/43671686/141850714-5889ba23-69c7-430a-8179-62c729521115.png)
est ok

Reply:
Spróbowałem z regionami US i UK - nic to nie daje

Reply:
Fixed